src/main/kotlin/org/jitsi/rtp/rtp/RtpPacket.kt

```diff
@@ -477,7 +479,10 @@ open class RtpPacket(
          * Resets this iterator back to the beginning of the extensions
          */
         internal fun reset() {
-            val extLength = extensionBlockLength - HeaderExtensionHelpers.TOP_LEVEL_EXT_HEADER_SIZE_BYTES
+            val extLength =
+                if (hasExtensions) extensionBlockLength - HeaderExtensionHelpers.TOP_LEVEL_EXT_HEADER_SIZE_BYTES
```

Comment:
This one is probably pretty negligible, since `extensionBlockLength` does a `hasExtensions` check itself.